package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "angular-material-css-vars",
-  "version": "1.1.1",
+  "version": "1.2.0",
```

```suggestion
  "version": "1.1.1",
```

Please don't change the version number.